jsonnet/prometheus-operator/prometheus-operator.libsonnet

```diff
@@ -85,9 +85,11 @@ function(params) {
           'prometheuses',
           'prometheuses/finalizers',
           'prometheuses/status',
+          'prometheuses/scale',
```

Comment:
I'm still not sure why we need to add "prometheuses/scale" and "prometheusagents/scale" to the operator's default permissions.

Reply:
Whoops, that was leftover from the previous commits! Good catch